### PR TITLE
Use default routegroup in sandboxes

### PR DESCRIPTION
**Pull request by scott-cotton**
note that some references from docs.signadot.com point to files which have now changed names
-- I'll work on that separately and merge them at the same time.

### CODE REVIEW

File: cli/sandbox-resource.yaml [hunk 1]
```diff
@@ -11,6 +11,10 @@ name: "@{dev}-@{team}-x-@{commit}"    # name of the sandbox
 spec:  
   cluster: staging                            # cluster in which this sandbox is to reside.
   description: "@{dev}'s work on feature-x"   # a short description
+  defaultRouteGroup:
```
Comment (daniel-de-vera) [inline cli/sandbox-resource.yaml:14]: Here we are adding the defaultRouteGroup endpoint but we are not removing the old forked endpoint from bellow:

```yaml
    endpoints:                                # endpoints associated with this fork.
    - name: customer                          # name of the endpoint.
      protocol: http                          # protocol (http or grpc).
      port: 8081                              # port to which the endpoint will resolve.
```

**Review by daniel-de-vera — COMMENTED** [review]
Apart from the previous comment, shouldn't we remove the nulls from here?: https://github.com/signadot/examples/blob/287de30a9816bc3ea6a41e803e888d7c0dcd22d5/cli/sandbox-template.yaml#L32

The rest LGTM, although I couldn't actually test it, it was just a visual check that I did (not sure if there are deployment files for running those examples).

**Review by daniel-de-vera — APPROVED** [review]
@scott-cotton, I'm approving the PR now, given the changes that I'm suggesting are small, and I don't want to block you.

**Comment by scott-cotton** [comment]
> @scott-cotton, I'm approving the PR now, given the changes that I'm suggesting are small, and I don't want to block you.

thanks!

**Pull request merged**